comp/otelcol/otlp/components/exporter/serializerexporter/logger.go

```diff
@@ -1,82 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2021-present Datadog, Inc.
-
-package serializerexporter
-
-import (
-	"fmt"
-
-	"go.uber.org/zap"
-)
-
-// zaplogger implements the tracelog.Logger interface on top of a zap.Logger
-type zaplogger struct{ logger *zap.Logger }
```

Comment:
You know why we used to need this, but don't anymore ?

Reply:
I added this before we exposed ZLogger in pkg/datadog in collector-contrib.